src/FocusTrap.ts

```diff
@@ -54,13 +55,25 @@ export const FocusTrap = defineComponent({
 
   setup(props, { slots, emit }) {
     let trap: FocusTrapI | null
-    const el = ref<HTMLElement | null>(null)
+    const el = ref<HTMLElement | ComponentPublicInstance | null>(null)
 
     const ensureTrap = () => {
       if (trap) {
         return
       }
 
+      if (el === null || el?.value === null) {
+        throw new Error(
+          '[focus-trap-vue] FocusTrap requires exactly one child.'
```

Comment:
it probably needs to check if there is one single child in `el.value`

Reply:
Quite confused about this point. I was under the impression that `el` is the (sole) child in `<focus-trap>`, so if `el` is `undefined`, then there simply is no children, and hence this message.

Perhaps the message can be changed to something like: "No child found. FocusTrap requires exactly one child." Just so that it's a little bit clearer.

Reply:
Unless Vue already warns against having multiple children when using ref (which shouldn't be the case when doing it on a multi root component), we will need a single element to pass to focus trap